tests/BlazingSQLTest/EndToEndTests/configOptionsTest.py

```diff
@@ -69,6 +69,20 @@ def main(dask_client, drill, spark, dir_data_file, bc, nRals):
     conf_opt_8["BLAZING_DEVICE_MEM_CONSUMPTION_THRESHOLD"] = 0.0001
     conf_opt_8["MEMORY_MONITOR_PERIOD"] = 5000000
 
+    # conf_opt_9
+    conf_opt_9 = {}
```

Comment:
Please make these conf_opt_9 and conf_opt_10 be part of other conf_option sets (2 and 3 would work). We dont want to add new ones because that means tests take longer to run